src/filesystem/impls/appshell/node/node_modules/fsevents_win/fsevents_win.cpp

```diff
@@ -337,7 +339,7 @@ DWORD WINAPI NodeFSEvents::Run(LPVOID lpData)
 	{
 		// iterate watching for directory changes until we're signalled to stop
 		BOOL bContinue = TRUE;
```

Comment:
Let's just get rid of `bContinue` it bothers me (both its name and its use in the `while` condition). Just like @bchintx pointed out, let's wrap the `ReadDirectory...` call in an if, this way @evgen48's concern of not handling the error condition will be addressed too.